### PR TITLE
WOR-887 fix azure connected tests

### PR DESCRIPTION
**Pull request by dvoet**
https://broadworkbench.atlassian.net/browse/WOR-887

### CODE REVIEW

File: service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/AzureControlledStorageContainerFlightTest.java [hunk 1]
```diff
@@ -113,46 +106,6 @@ public void createAndDeleteAzureStorageResource() throws InterruptedException {
     verifyStorageAccountContainerIsDeleted(storageContainerName);
   }
 
-  @Disabled("TODO(TOAZ-286): Re-enable this test when the ticket is fixed")
-  @Test
-  public void
-      createAzureStorageContainerFlightFailedBecauseLandingZoneDoesntHaveSharedStorageAccount()
```
Comment (dvoet) [inline service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/AzureControlledStorageContainerFlightTest.java:119]: This edge case is covered by a unit test

**Review by cahrens — APPROVED** [review]

**Review by marctalbott — APPROVED** [review]
